scala/scala-impl/resources/inspectionDescriptions/ConvertExpressionToSAM.html

```diff
@@ -1,21 +1,15 @@
 <html>
 <body>
-Checks if an expression can be converted to SAM (Single Abstract Method).
-
-<p>
-Before:
-<pre>
+Reports if an expression that can be converted to SAM (Single Abstract Method).
```

Comment:
Reports expressions that can be converted to SAM (Single Abstract Method)  types.